modules/windows-machineset-vsphere.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * windows_containers/adding-windows-workloads.adoc
+
+[id="windows-machineset-vsphere_{context}"]
+= Sample YAML for a Windows MachineSet on vSphere
+
+This sample YAML defines a Windows MachineSet running on VMware vSphere that
+the Windows Machine Config Operator (WMCO) can react upon.
+
+_WIP_
```

Comment:
same question, merge as-is or exclude for now?

Reply:
Same as above; I'll remove if this does not get supported for the initial release.